guarddog/cli.py

```diff
@@ -165,14 +165,18 @@ def _scan(identifier, version, rules, exclude_rules, output_format, exit_non_zer
     if scanner is None:
         sys.stderr.write(f"Command scan is not supported for ecosystem {ecosystem}")
         exit(1)
-    results = {}
+    results = []
```

Comment:
this is going to be a breaking change of the output format, correct?

Reply:
No, nothing will change when you scan only a package

Reply:
Seems like it's changing the output format, so this would be a breaking chance. Can we avoid it?

```
$ git checkout main
$ python -m guarddog pypi scan /tmp/requests.tar.gz --output-format json
{"results": {"download-executable": {}, "npm-silent-process-execution": {}, "obfuscation": {}, "steganography": {}, "npm-install-script": {}, "exfiltrate-sensitive-data": {}, "npm-exec-base64": {}, "exec-base64": {}, "code-execution": {}, "shady-links": {}, "npm-serialize-environment": {}, "silent-process-execution": {}, "cmd-overwrite": {}}, "errors": {}, "issues": 0}
```

v.s.

```
$ gh pr checkout 222
$ python -m guarddog pypi scan /tmp/requests.tar.gz --output-format json
{"package": "/tmp/requests.tar.gz", "results": {}, "errors": {}, "issues": 0}
```

Reply:
Fixed!